src/components/AxisCollection/CollapsedAxis.js

```diff
@@ -36,6 +38,7 @@ export default class CollapsedAxis extends Component {
     offsetx: 0,
     onMouseEnter: null,
     onMouseLeave: null,
+    yAxisPlacement: AxisPlacement.RIGHT,
```

Comment:
ANY REASON WE HAVE THIS TO THE RIGHT AS DEFAULT????

Reply:
JUST KEEPING IT CONSISTENT!!!11!